horovod/spark/torch/remote.py

```diff
@@ -54,13 +52,21 @@ def RemoteTrainer(estimator, metadata, last_checkpoint_state, run_id, dataset_id
     user_verbose = estimator.getVerbose()
     train_minibatch_fn = estimator.getTrainMinibatchFn()
     train_minibatch = train_minibatch_fn if train_minibatch_fn else _train_minibatch_fn()
+    loss_fns_pre_train = to_list(estimator.getLoss(),
```

Comment:
Let's pull `num_labels` out from line 63 and put it just above line 55 (`loss_fns_pre_train = ...`).  That way, we can just reference `num_labels` instead of `len(label_columns)`.